cylc/flow/workflow_files.py

```diff
@@ -1331,6 +1331,11 @@ def validate_workflow_name(name: str) -> None:
             "Workflow name cannot be a path that points to the cylc-run "
             "directory or above"
         )
+    if re.findall(r'^run(N|\d+)$', Path(name).name):
```

Comment:
@MetRonnie  - If this doesn't suite then I think this is easier resolved talking on the phone. I've changed the condition not the message because I'm not sure I want to exclude potentially legit flow names like:
- underrun2
- roadrunNer-beep-beep 
Should be ok. Having said which if you think we should ban any name containing the expression I don't think I can think of any non contrived examples actually likely to cause problems. (https://www.thefreedictionary.com/words-containing-run)